src/main/flight/imu.c

```diff
@@ -165,6 +165,15 @@ STATIC_UNIT_TESTED void imuComputeRotationMatrix(void)
 #endif
 }
 
+#ifdef USE_WING
+// Angle in between nose axis of the craft and the horizontal plane in ground reference.
+// Poitive angle - nose up, negative angle - nose down.
+float getNoseAngleSin(void)
```

Comment:
it's not exactly pitch, because normally pitch is in the craft coordinate system.
For example you can roll 90 degrees, and then pitch up and down. The "nose angle" will be zero. But pitch will be changing.

I'll make in consistent with `getCosTiltAngle`

Reply:
Not in betaflight. In body frame, rotations are applied in yaw/pitch/roll (around z-y'-x'' body axes) order. So with roll=90deg, elevator will change yaw, not pitch.

Also, in BF, positive pitch means nose down (https://github.com/betaflight/betaflight/blob/master/src/main/flight/imu.c#L317). It is better to stick to this convention. Coordinate systems are already complicated enough in BF.